### PR TITLE
[5.2] Media manager fix default adapter handling

### LINKED ISSUE #44277: Media manager error, when "Path to Files Folder" field contain folder that not in the plugin "FileSystem - Local", COM_MEDIA_ERROR_ACCOUNT_NOT_FOUND
### Steps to reproduce the issue
If the standard image directory is changed under Plugins System File System, TinyMCe produces an error message
COM_MEDIA_ERROR_ACCOUNT_NOT_FOUND

Please fix it. Thanks

### PR DESCRIPTION
**Pull request by Fedik**
Pull Request for Issue https://github.com/joomla/joomla-cms/issues/44277 .

### Summary of Changes

Media manager produces  an error when "Path to Files Folder" field contain folder that not in the plugin "FileSystem - Local".
I have fixed fallback to first available adapter.

### Testing Instructions

Edit Media manager global options, set "Path to Files Folder" to some random existing folder.
Make sure this folder does not selected in plugin "FileSystem - Local".
Enable Drag&Drop upload for TinyMCE.
Open Article editing.

### Actual result BEFORE applying this Pull Request
An error page

### Expected result AFTER applying this Pull Request
No error

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed
- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

**Comment by spike277** [comment]
Solution only works in backend. Frontend-Editing have still the Error

**Comment by Fedik** [comment]
> Frontend-Editing have still the Error

Cannot be, the code is the same for both, for administrator and for site.

File: administrator/components/com_media/src/Provider/ProviderManagerHelperTrait.php [hunk 1]
```diff
@@ -148,14 +149,20 @@ protected function getDefaultAdapterName(): ?string
             return $this->defaultAdapterName;
         }
 
-        $defaultAdapter = $this->getAdapter('local-' . ComponentHelper::getParams('com_media')->get('file_path', 'images'));
+        try {
+            // Check for default path in Media config, and whether associated account exists,
+            // and use it as default adapter.
```
Comment (laoneo) [inline administrator/components/com_media/src/Provider/ProviderManagerHelperTrait.php]: Can you comment it with /* */?
https://developer.joomla.org/coding-standards/inline-code-comments.html

**Comment by bcordis** [comment]
I have tested this item :white_check_mark: successfully on b80e9eb080f146b885f64d595a3653e8b8867984<br /><br />Tested Successfully<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/44313">issues.joomla.org/tracker/joomla-cms/44313</a>.</sub>

**Comment by viocassel** [comment]
I have tested this item :white_check_mark: successfully on b80e9eb080f146b885f64d595a3653e8b8867984<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/44313">issues.joomla.org/tracker/joomla-cms/44313</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/44313">issues.joomla.org/tracker/joomla-cms/44313</a>.</sub>

**Comment by spike277** [comment]
Yes, me to. Problem in Frontend still in Joomla 5.2.1. Not good :( Hope also to fix it.

**Pull request merged**